src/main/java/io/github/thebusybiscuit/cscorelib2/recipes/MinecraftRecipe.java

```diff
@@ -108,7 +109,12 @@
 		recipe -> new RecipeChoice[] {recipe.getInputChoice()}, (input, stream) ->
 		stream.filter(recipe -> recipe.getInputChoice().test(input[0])).findAny().map(StonecuttingRecipe::getResult)
 	);
-	
+
+	public static final MinecraftRecipe<SmithingRecipe> SMITHING_RECIPE = new MinecraftRecipe<>(Material.SMITHING_TABLE, SmithingRecipe.class, recipe -> recipe.length == 2,
```

Comment:
I told you to defer the class loading, as this would cause versions without the SmithingRecipe.class to fail class initialization

Reply:
reeee